socialhome/users/tests/test_models.py

```diff
@@ -102,3 +103,14 @@ def test_get_last_name(self):
         assert profile.get_last_name() == ""
         profile.name = ""
         assert profile.get_last_name() == ""
+
+    def test_content_saved_in_correct_order(self, admin_client, rf):
```

Comment:
This is in `socialhome/users`. It should be in `socialhome/content` app `TestContentModel`.